package/utils/jboot-tools/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jboot-tools
```

Comment:
Reading only the package name I think the `jboot-tools` is related to the bootloader of the router.
Sure, the description clearly specifies that it will poke around the config partition.

How about putting `eeprom` in the name: `jboot-eeprom-tools`? Unsure about `fw`, `config` and others.

Reply:
eeprom-tools doesn't really fit. It is similar to the uboot-envtools and fritz-tffs. But to be honest I don't really care about the name.

Reply:
>But to be honest, I don't really care about the name.

Ok, then keep `jboot-tools`' name as is. :)

Reply:
Ok, then keep this as is.